test/datagenerator/1800medicare.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+type field struct {
+	l int
+	v string
+}
+
+type record struct {
+	hicn, blk                                    field
+	firstName, middleName, lastName              field
+	dob                                          field
+	addr1, addr2, addr3, city, state, zip5, zip4 field
+	gender                                       field
+	encDate                                      field
+	effDate, srcCode, mechCode, prefInd          field
+	saEffDate, saSrcCode, saMechCode, saPrefInd  field
+	acoID, acoName                               field
+}
+
+func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
+	now := time.Now()
+
+	// 1-800-MEDICARE filename format: (T|P)#EFT.ON.ACO.NGD1800.DPRF.Dyymmdd.Thhmmsst
+	fileName := fmt.Sprintf("T#EFT.ON.ACO.NGD1800.DPRF.%s", now.Format("D060102.T1504050"))
+	outf, err := os.Create(fileName)
+	if err != nil {
+		panic(err)
+	}
+
+	_, err = outf.WriteString(fmt.Sprintf("HDR_BENEDATASHR%s\n", now.Format("20060102")))
+	if err != nil {
+		panic(err)
+	}
+
+	for _, acoID := range []string{"A9990", "A9991", "A9992", "A9993", "A9994"} {
+		inf, err := os.Open(fmt.Sprintf("%s-HICNs", acoID))
```

Comment:
Is this creating 0-5 suppression records for each hicn found from the synthetic data files A9990 --> A9994?

Reply:
0-5 records for about half of the HICNs in those files, based on this line: https://github.com/CMSgov/bcda-app/pull/366/files#diff-31f06376cac09449837e97f31ce0172fR55

Reply:
ahhh gotcha, I was confused on what that line was doing. Got it, slick solution for that.